pkgs/applications/networking/feedreaders/castget/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, glib, ronn, curl, id3lib, libxml2 }:
```

Comment:
This style is pretty unconventional. If you have a multi-line arglist, you usually put the final `}:` on a separate line. Personally I prefer to just make one line per dependency, but there is no clear consensus on that.

Reply:
Fixed.